repos/c2sm/packages/py-poetry-core/package.py

```diff
@@ -0,0 +1,43 @@
+from spack.package import *
+from spack.pkg.builtin.py_poetry_core import PyPoetryCore as SpackPyPoetryCore
+
+
+# TODO (magdalena) mostly copied from https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/py-poetry-core/package.py
```

Comment:
Still needed?
Spack has now version 0.20.1

Reply:
Perhaps not needed, but Magdalena had added a recent version (1.6.1). Will test with inbuilt.